packages/release/release.spec

```diff
@@ -136,6 +136,7 @@ Requires: %{_cross_os}findutils
 Requires: %{_cross_os}glibc
 Requires: %{_cross_os}grep
 Requires: %{_cross_os}grub
+Requires: %{_cross_os}hwloc
```

Comment:
> Add the binary package to release to include in all variants since EFA can also use hwloc for finding colocation-relevant data.

You're saying that EFA (the kernel driver) does an upcall to `hwloc` in the host OS? Or what exactly does that dependency look like?

I'd prefer to make this NVIDIA-only if there's nothing in the host that actually uses this.

Reply:
I went to find the specific references to these to trim them down and I'm not finding direct usage of `hwloc-ls`. The errors I first encountered are referencing [this code](https://github.com/open-mpi/ompi/blob/v4.1.8/orte/mca/rtc/hwloc/rtc_hwloc.c#L424-L447) which has a library dependency on hwloc. In light of not finding any specifics, I think we should avoid adding this to images by default, and use this more as a debugging package when needed.